File: src/main/java/com/elf/vipForElf/web/dto/RealEstateAgencyListDTO.java
package com.elf.vipForElf.web.dto;

import com.elf.vipForElf.domain.realEstateAgency.vo.RealEstateAgencyRow;
import lombok.Data;

import java.util.List;

@Data
public class RealEstateAgencyListDTO {
    Long totalCount;
    List<RealEstateAgencyRow> realEstateAgencyRowList;

    public RealEstateAgencyListDTO(Long totalCount, List<RealEstateAgencyRow> realEstateAgencyRowList){
        this.totalCount = totalCount;
        this.realEstateAgencyRowList = realEstateAgencyRowList;
    }
}

File: src/main/java/com/elf/vipForElf/web/dto/RealEstateAgencyListDTO.kt
package com.elf.vipForElf.web.dto

import com.elf.vipForElf.domain.realEstateAgency.vo.RealEstateAgencyRow

data class RealEstateAgencyListDTO(
    val totalCount: Long?,
    val realEstateAgencyRowList: List<RealEstateAgencyRow>?
)
